README.md

```diff
@@ -1021,8 +1021,6 @@ Look at this [issue](https://github.com/xmartlabs/Eureka/issues/96).
 * Set up a new header/footer data ....
 
 ```swift
-section.header = "Header Title" // use string literal as a header/footer data. HeaderFooterView conforms to ExpressibleByStringLiteral.
```

Comment:
Actually you can create a `HeaderFooterView` instance from a string literal. So, maybe the deleted code can be replaced with:

```swift
let header: HeaderFooterView<UIView> = "Header Title" // use string literal as a header/footer data. HeaderFooterView conforms to ExpressibleByStringLiteral.
section.header = header
```

What do you think?